src/Raven.Server/Documents/Subscriptions/Stats/SubscriptionBatchStatsScope.cs

```diff
@@ -33,10 +33,18 @@ public void RecordBatchInfo(long taskId, string taskName, long connectionId, lon
             _stats.BatchId = batchId;
         }
         
-        public void RecordDocumentsInfo(long documentsCount, long documentsSize)
+        public void RecordDocumentsInfo(long documentsCount, long documentsSize,
```

Comment:
I wonder if we should split that into few methods? Right now we have one huge with 6 params. 

what if do:
ReportIncludesInfo(includesCount, includesSize)
ReportDocumentsInfo(docsCount, docsSize).
etc?

Reply:
done